inlong-dashboard/src/metas/streams/common/StreamDefaultInfo.ts

```diff
@@ -105,6 +105,10 @@ export class StreamDefaultInfo implements DataWithBackend, RenderRow, RenderList
           label: 'CSV',
           value: 'CSV',
         },
+        {
+          label: 'Key-Value',
+          value: 'Key-Value',
```

Comment:
Should the value communicated with the manager be `KV` here?
Such as: 
https://github.com/apache/inlong/blob/45d98689f146431258eefdd9148cde64daf9a020/inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/stream/InlongStreamRequest.java#L61

Reply:
We need to confirm how to communicate with the manager.

Reply:
It is found that the Manager does not support it yet.
Suggest using `KV` when communicating with the Manager.